cmd/multi_cmd.go

```diff
@@ -14,25 +14,37 @@ import (
 var multiCmd = cobra.Command{
 	Use:  "multi",
 	Long: "Start multi-tenant API server",
-	Run: func(cmd *cobra.Command, args []string) {
-		execWithConfig(cmd, multi)
-	},
+	Run:  multi,
 }
 
-func multi(config *conf.GlobalConfiguration) {
-	db, err := models.Connect(config)
+func multi(cmd *cobra.Command, args []string) {
+	globalConfig, err := conf.LoadGlobal(configFile)
+	if err != nil {
+		logrus.Fatalf("Failed to load configuration: %+v", err)
+	}
+	if globalConfig.OperatorToken == "" {
+		logrus.Fatal("Operator token secret is required")
+	}
+	if globalConfig.DB.Namespace != "" {
```

Comment:
that isn't required right? The namespace corresponds to the name of the tables, which can be empty. We can use a single set of tables regardless of tenancy